src/vue/components/inflections-table-wide.vue

```diff
@@ -202,6 +209,51 @@ export default {
       if (this.state.view && this.state.view.isImplemented) {
         this.state.view.wideView.collapsed = this.state.collapsed
       }
+
+      if (this.app.platform.isMobile && scrollPos) {
+        this.checkAndFixScroll(scrollPos.parentIBNode, scrollPos.scrollTop)
+      }
+    },
+
+    findCurrentScrollPos() {
+      let maxSteps = 8
+      let parentIBNode = null
+      let currentParent = null
+      let scrollTop = null
+
+      while (maxSteps >= 0 && !parentIBNode) {
+        maxSteps--
+        currentParent = !currentParent ? this.$el.parentNode : currentParent.parentNode
+        if (currentParent && currentParent.classList && currentParent.classList.contains('alpheios-panel__content')) {
```

Comment:
Why not just query for the element with the .alpheios-panel__content class directly? What do we get from walking up the parents to it?

Reply:
It is done only to garantee that we well get the appropriate panel content block that is currently scrolled. 
We could have several blocks as it is a class.
And it would be lighter operation than to go through all nodes searching for the class in the document.